native_client/deepspeech.h

```diff
@@ -128,8 +98,77 @@ namespace DeepSpeech
       char* stt(const short* aBuffer,
                 unsigned int aBufferSize,
                 int aSampleRate);
+
+      /**
+       * @brief Setup a context used for performing streaming inference.
+       *        the context pointer returned by this function can then be passed
+       *        to {@link feedAudioContent()} and {@link finishStream()}.
+       *
+       * @param aPreAllocFrames Number of timestep frames to reserve. One timestep
+       *                        is equivalent to two window lenghts (50ms), so
```

Comment:
Is this dependant on parameters set during the training ?

Reply:
It's dependent on the training code, yes. Feature computation for training and inference need to be consistent.

Reply:
Maybe it's worth mentionning it?